spotbugsTestCases/src/java/sfBugs/Bug3458246.java

```diff
@@ -0,0 +1,37 @@
+package sfBugs;
+
+import java.util.logging.Logger;
```

Comment:
Is SLF4J fixed yet? There's supposed to be a Gradle fix.

Reply:
It is old test code, not related with that trouble.